docs/tutorials/ionq/getting_started.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<p>"Above the"  -&gt; "Above, the"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>reworded</p>